docs/pages/api-reference/liveblocks-node.mdx

```diff
@@ -273,7 +275,681 @@ export default async function auth(req, res) {
 }
 ```
 
-### Liveblocks.getThreads [#get-threads]
+### Room
+
+#### Liveblocks.getRooms [#get-rooms]
+
+Returns a list of rooms that are in the current project. The project is
+determined by the secret key you’re using. Rooms are sorted by creation time,
+with the newest room at index `0`. This is a wrapper around the
+[Get Rooms API](/docs/api-reference/rest-api-endpoints#get-rooms) and returns
+the same response.
+
+```ts
+const { data: rooms, nextPage } = await liveblocks.getRooms();
+
+// A list of rooms
+// [{ type: "room", id: "my-room-id", ... }, ...]
+console.log(rooms);
+
+// A pagination value used for retrieving the next page of results
+// "L3YyL3Jvb21z..."
+console.log(nextPage);
+```
+
+A number of options are also available, enabling you to filter for certain
+rooms.
+
+```ts
+const { data: rooms, nextPage } = await liveblocks.getRooms({
+  // Optional, the amount of rooms to load, between 1 and 100, defaults to 20
+  limit: 20,
+
+  // Optional, filter for rooms that allow entry to group ID(s) in `groupAccesses`
+  groupIds: ["engineering", "design"],
+
+  // Optional, filter for rooms that allow entry to a user's ID in `usersAccesses`
+  userId: "my-user-id",
+
+  // Optional, filter for rooms with custom metadata in `metadata`
+  metadata: {
+    myRoomType: "whiteboard",
+  },
+
+  // Optional, cursor used for pagination, use `nextPage` from the previous page's response
+  startingAfter: "L3YyL3Jvb21z...",
+});
+```
+
+You can use `nextPage` to paginate rooms. In this example, when `getNextPage` is
+called, the next set of rooms is added to `pages`.
+
+```ts
+import { RoomInfo } from "@liveblocks/node";
+
+// An array of pages, each containing a list of retrieved rooms
+const pages: RoomInfo[][] = [];
+
+// Holds the pagination cursor for the next set of rooms
+let startingAfter;
+
+// Call to get the next page of rooms
+async function getNextPage() {
+  const { data, nextPage } = await liveblocks.getRooms({ startingAfter });
+  pages.push(data);
+  startingAfter = nextPage;
+}
+```
+
+#### Liveblocks.createRoom [#post-rooms]
+
+Programmatically creates a new room from a room ID. The `defaultAccesses` option
+is required—setting it to `["room:write"]` creates a public room, whereas
```

Comment:
Should the `-` have a space before and after?

Reply:
This an em dash, and they're mostly used [without surrounding spaces](https://practicaltypography.com/hyphens-and-dashes.html#:~:text=Em%20and%20en%20dashes%20are%20typ%C2%ADi%C2%ADcally%20set%20flush%20against%20the%20sur%C2%ADround%C2%ADing%20text.%20Some%20fonts%20in%C2%ADclude%20a%20lit%C2%ADtle%20white%20space%20around%20the%20em%20dash%3B%20some%20don%E2%80%99t.%20If%20your%20em%20dashes%20look%20like%20they%E2%80%99re%20be%C2%ADing%20crushed%E2%80%94par%C2%ADtic%C2%ADu%C2%ADlarly%20if%20you%E2%80%99re%20set%C2%ADting%20type%20on%20screen%E2%80%94it%E2%80%99s%20fine%20to%20add%20word%20spaces%20be%C2%ADfore%20and%20after.), but you can add them if your text feels crushed. I agree, this doesn't look right here, I'll change it!

If you're interested in Typography, or would like to check best practices when writing, I super recommend [Practical Typography](https://practicaltypography.com/). I always try to follow their advice 🙂